### ISSUE TITLE
Feature request: not only Google calendar, but CalDav as well

### ISSUE THREAD

**Issue by DoctorSubtilis** [Zren/plasma-applet-eventcalendar#206]
An open-source program, in my opinion, should give the possibility to sync non only with Google calendar, but with a CalDav as well (that is fully open-source, unlike Google).  
Thanks!

**Comment by Termuellinator** [comment]
you can do that already (you have to activate the module) - it would be nice to be able to use that as a doubleclick-action, too though ^^

**Comment by DoctorSubtilis** [comment]
Ops... Thanks, but what module? And how can I activate it?

**Comment by Termuellinator** [comment]
I can't check right now, but iirc it's called something with "PIM". This let's you check the calendars you have setup in akonadi/korganizer

**Comment by DoctorSubtilis** [comment]
Uhm, so far I don't have akonadi installed. It's necessary?

**Comment by Termuellinator** [comment]
i think so - but don't quote me on that ^^

**Comment by DoctorSubtilis** [comment]
but, after installing korganizer you are sure I'll be able to use event-calendar with a caldav?

**Comment by Termuellinator** [comment]
At least it works for me, yes - at least for viewing the existing entries, creating new entries with doubleclick still tries to open google

**Comment by DoctorSubtilis** [comment]
I have installed Korganizer, and reset PC, but don't see a way to set a caldav...

**Comment by Termuellinator** [comment]
Now i'm back on my PC to check. You can activate the "PIM-Appointment-Module" (? or however the german "PIM-Terminmodul" is named in your language ^^) in the "Events" Tab: https://i.imgur.com/a3AKAwI.png

After that there is a new entry for the PIM module on the left where you can select the calendars that are set in akonadi:
https://i.imgur.com/mJVt4Kt.png

**Comment by DoctorSubtilis** [comment]
Maybe we have two different Linux distro: I have KDE Neon (with packages from Ubuntu 20.04)... and this is why I don't see what you see ...

**Comment by Termuellinator** [comment]
Good catch with the PIM-Plugins, haven't thought about that being a dependency!
My Caldav is at mailbox.org, but local works, too i think (at least i can select the local ones, i don't use them though). Do you see your calendar in KOrganizer?

**Comment by DoctorSubtilis** [comment]
Yes. I see, it's a Nextcloud caldav.

**Comment by Zesko** [comment]
KDE PIM `kdepim-addons` does not work in Manjaro or Arch Linux after update. There is no KDE PIM in KDE NEON with latest version stable.

I think, it should add new feature to support CalDAV (independence of KDE-PIM)

**Comment by Zesko** [comment]
Or it would be nice if it supports Thunderbird Calendar integration.

**Comment by KenwoodFox** [comment]
Trying this out on kde-neon, was able to install kdepim-addons

**Comment by Rik44444** [comment]
+1 for Nextcloud / CalDav
